queries/yuck/highlights.scm

```diff
@@ -0,0 +1,66 @@
+(ERROR) @error
+
+(line_comment) @comment
+
+; keywords and symbols
+
+(keyword) @keyword
+(symbol) @function
+
+; literals
+
+(bool_literal) @constant.builtin.boolean
```

Comment:
` @boolean`. Do other editors use `@constant.builtin.boolean`?

Reply:
Well I guess this is yet another tag that differs between helix (my primary editor currently) and neovim.